code/workspaces/web-app/src/components/stacks/StackCards.js

```diff
@@ -5,11 +5,11 @@ import StackCard from './StackCard';
 import NewStackButton from './NewStackButton';
 import PermissionWrapper from '../common/ComponentPermissionWrapper';
 
-const breakPoints = { xs: 12, sm: 12, md: 6, lg: 4 };
+const breakPoints = { xs: 12 };
 
 const StackCards = ({ stacks, typeName, openStack, deleteStack, editStack, openCreationForm, userPermissions,
   createPermission, openPermission, deletePermission, editPermission }) => (
-  <Grid container spacing={2}>
+  <Grid container spacing={0}>
```

Comment:
Not sure the Grid is necessary any more. Might be clearer what is going on if it was just wrapped in a div with
```
display: flex;
flex-direction: column;
```

Reply:
Initially kept it as a grid in case we wanted to revert, but agree best to change.  Thanks, done.